awslogs/core.py

```diff
@@ -42,7 +49,7 @@ def __init__(self, **kwargs):
         self.log_stream_name = kwargs.get('log_stream_name')
         self.filter_pattern = kwargs.get('filter_pattern')
         self.watch = kwargs.get('watch')
-        self.color_enabled = kwargs.get('color_enabled')
+        self.color_enabled = COLOR_ENABLED.get(kwargs.get('color'), True)
```

Comment:
```suggestion
        self.color_enabled = COLOR_ENABLED.get(kwargs.get('color'), 'auto')
```
I imagine it'll require more changes to tests, but it'd reflect the CLI

Reply:
I don't think that change is correct. Note that the `True` default is for the call of `COLOR_ENABLED.get`, not `kwargs.get`.

The `ArgumentParser` should ensure that `kwargs` always contains `color`, defaulted to `auto` (see `bin.py:129`), so the `.get` call shouldn't strictly be necessary - I used it since all the surrounding code does too.

The `ArgumentParser` also enforces that `--color` is one of `['never', 'always', 'auto']`, ie running `awslogs --color=off` will error out before it even gets to this point in the code.

I suppose neither of the `.get` calls is strictly necessary, but I think it's good to be defensive.

Reply:
I think @apinkney97 is right on this one. That being said, I would rather directly access kwargs instead of using get, but I get your point about being defensive.